src/layers/domain/core/product.py

```diff
@@ -1,98 +1,132 @@
-"""
-The collection of models required to create Products:
-
-OdsOrganisation -creates-> ProductTeam -creates-> Product
-                ∟ emits -> ProductTeamCreatedEvent
-"""
-from typing import Iterable, TypeVar
+from enum import Enum
 from uuid import UUID
 
-from domain.events.event import Event
+from domain.core.aggregate_root import AggregateRoot
```

Comment:
possible to add type annotations to the file?

Reply:
Added lots.